src/visualization/write2vtk.jl

```diff
@@ -185,12 +185,12 @@ end
 
 """
     trixi2vtk(initial_condition::InitialCondition; output_directory="out",
-              prefix="", filename="coordinates", custom_quantities...)
+              prefix="", filename="initial_condition", custom_quantities...)
 
-Convert `InitialCondition` data to VTK format.
+Convert [`InitialCondition`](@ref) data to VTK format.
 
 # Arguments
-- `initial_condition::InitialCondition`: [Initial condition](@ref InitialCondition) to be saved.
+- `initial_condition::InitialCondition`: [`InitialCondition`](@ref) to be saved.
```

Comment:
```suggestion
- `initial_condition`: [`InitialCondition`](@ref) to be saved.
```
Redundant. It's already mentioned after the colon.